.travis.yml

```diff
@@ -30,12 +31,17 @@ matrix:
       env: WP_VERSION=5.2 WP_MULTISITE=1
     - php: 7.3
       env: WP_VERSION=latest WP_MULTISITE=1
-    - php: 7.3
+    - php: 7.4snapshot
```

Comment:
We have a few more references to the snapshot version in the yaml and also the version tested against is 5.2 which is not php 7.4 compatible. We may need to use 5.3 as our target version or CI will fail and not allow us to merge/publish this release. There are two other references to the 7.4 snapshot and 5.2 WordPress.